components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -746,6 +746,15 @@ export const RepositoryNotFound: FC<{ error: StartWorkspaceError }> = ({ error }
             />
         );
     }
+    if (authProvider.id.toLocaleLowerCase() === "public-github") {
+        return (
+            <RepositoryInputError
+                title={`Although you appear to have the correct authorization credentials, the '${owner}' organization has enabled OAuth App access restrictions, meaning that data access to third-parties is limited. For more information on these restrictions, including how to enable this app, visit https://docs.github.com/articles/restricting-access-to-your-organization-s-data/.`}
+                linkText="Check Organization Permissions"
+                linkHref={"https://github.com/settings/connections/applications/484069277e293e6d2a2a"}
```

Comment:
Will the application always be this? Even in Enteprise? Maybe we want to check `isGitpodIo`

Reply:
added